src/brisk.jl

```diff
@@ -0,0 +1,100 @@
+type BRISK{S, T, O} <: Params
+    threshold::Float64
+    octaves::Int
+    pattern_scale::Float64
+    pattern_table::Vector{Vector{S}}
+    smoothing_table::Vector{T}
+    orientation_weights::Vector{O}
+    short_pairs::Vector{O}
+    long_pairs::Vector{O}
+end
+
+typealias OrientationPair Vector{Int}
+typealias SamplePair Vector{Float64}
+
+function BRISK(; threshold::Float64 = 0.25, octaves::Int = 4, pattern_scale = 1.0)
```

Comment:
The `threshold` and `octaves` are for scale invariance. They will be used once AGAST is added.